### PR TITLE
Updated recommended Brave configuration

### PR DESCRIPTION
**Pull request by 5-tom**
Changes proposed in this PR:

- Updated recommended Brave configuration
- ~~Warning about Brave's Google Play Services dependency~~ https://github.com/privacyguides/privacyguides.org/pull/2561#discussion_r1587038135

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

**Comment by redoomed1** [comment]
> * Warning about Brave's Google Play Services dependency

Please see the following update: https://discuss.privacyguides.net/t/latest-brave-for-android-has-a-hard-dependency-on-google-play-services-microg/18085/30

File: docs/mobile-browsers.md [hunk 1]
```diff
@@ -113,16 +123,24 @@ Brave allows you to select additional content filters within the internal `brave
...
+- [ ] Uncheck **IPFS Gateway** (2)
 - [x] Select **Close tabs on exit**
 - [ ] Uncheck **Allow privacy-preserving product analytics (P3A)**
 - [ ] Uncheck **Automatically send diagnostic reports**
 - [ ] Uncheck **Automatically send daily usage ping to Brave**
 
 </div>
 
-1. InterPlanetary File System (IPFS) is a decentralized, peer-to-peer network for storing and sharing data in a distributed filesystem. Unless you use the feature, disable it.
+1. Safe Browsing communicates with Google servers in order to download blacklists.
```
Comment (redoomed1) [inline docs/mobile-browsers.md:136]: It's worth noting the following:

> Android devices without Google Play Services are currently unable to enable Safe Browsing in Brave.

Source: https://support.brave.com/hc/en-us/articles/15222663599629-Safe-Browsing-in-Brave

File: docs/mobile-browsers.md [hunk 2]
```diff
@@ -66,6 +66,13 @@ Brave is built upon the Chromium web browser project, so it should feel familiar
 
 </div>
 
+<div class="admonition warning" markdown>
+<p class="admonition-title">Warning</p>
+
+Brave for Android has a hard dependency on Google Play Services, and as a result will not work on [AOSP derivatives](https://www.privacyguides.org/en/android/#aosp-derivatives) without additional tweaks, such as [Sandboxed Google Play](https://grapheneos.org/features#sandboxed-google-play) on GrapheneOS.
```
Comment (redoomed1) [inline docs/mobile-browsers.md:72]: There is already a merged PR that fixed this regression (you can find it in the forum post I linked previously), so this section shouldn't be necessary.

**Review by redoomed1 — CHANGES_REQUESTED** [review]

File: docs/mobile-browsers.md [hunk 3]
```diff
@@ -66,6 +66,13 @@ Brave is built upon the Chromium web browser project, so it should feel familiar
...
+
+</div>
+
```
Comment (jonaharagon) [inline docs/mobile-browsers.md:75]:
```suggestion
```

**Review by jonaharagon — CHANGES_REQUESTED** [review]
Thanks for the PR!

As noted by @redoomed1, we don't want to jump the gun with adding warnings about issues that are ultimately temporary, unless they are long lasting or recurring problems. In this case it was an upstream issue that Brave is fixing.

Other changes LGTM at a glance, I'll check them out on my phone in a bit. Not sure about the safe browsing one. I think there was a previous discussion about the safe browsing setting already, but I'm not sure... I'll look for it.

**Review by redoomed1 — CHANGES_REQUESTED** [review]
Regarding Safe Browsing, here are some relevant excerpts and links to documentation:

- [Brave's privacy policy wrt SB](https://brave.com/privacy/browser/#safe-browsing)
  <details>
  <summary>Relevant excerpt</summary>

  > On Android, we use the [SafetyNet Safe Browsing API](https://developer.android.com/training/safetynet/safebrowsing) which sends partial URL hashes directly to Google when a URL is determined to be potentially malicious by the list stored locally on your device, as per the Safe Browsing Update API.
</details>

- [Brave's official documentation on their implementation of SB](https://support.brave.com/hc/en-us/articles/15222663599629-Safe-Browsing-in-Brave)
  <details>
  <summary>Relevant excerpts</summary>

  ### In the "Android" section
  > the Android version of Brave makes use of a service provided by the operating system. Specifically, it uses the SafetyNet Google Play API and on-device lists that are shared between all of the applications performing Safe Browsing checks. _Android devices without Google Play Services are currently unable to enable Safe Browsing in Brave_.

  > any requests from the operating system service to the Safe Browsing server (whether they originate from the Brave application or not) are done directly and do not go through a Brave proxy. This means that your IP address may be seen (and logged) by Google.
</details>

- [Developer documentation on the SafetyNet Safe Browsing API](https://developer.android.com/privacy-and-security/safetynet/safebrowsing#data-collected) (referenced in the previous excerpts)
- [Developer documentation on the Safe Browsing Update API](https://developers.google.com/safe-browsing/v4/update-api) (referenced in Brave's Safe Browsing docs)
  <details>
  <summary>Relevant excerpts</summary>

  > The Update API lets your client applications download hashed versions of the Safe Browsing lists for storage in a local database. URLs can then be checked locally. Only if a match is found in the local database does the client need to send a request to the Safe Browsing servers to verify whether the URL is included on the Safe Browsing lists.

  In the "[Checking URLs](https://developers.google.com/safe-browsing/v4/update-api#checking-urls)" section
  > At no point does Google learn about the URLs you are examining. Google does learn the hash prefixes of URLs, but the hash prefixes don’t provide much information about the actual URLs.
</details>

---
This is all to say that the footnote under the optional suggestion about Safe Browsing should reference official docs, and offer more clarity about Brave's implementation of SB on _specifically Android_. I find the current footnote somewhat ambiguous.

I might draft one later if I have some time.

File: docs/mobile-browsers.md (hunk 1 above)
Comment (redoomed1) [inline docs/mobile-browsers.md:136]:
```suggestion
1. Brave's [implementation of Safe Browsing](https://support.brave.com/hc/en-us/articles/15222663599629-Safe-Browsing-in-Brave) on Android **does not** send [Safe Browsing network requests](https://developers.google.com/safe-browsing/v4/update-api#checking-urls) through a Brave-operated proxy like its desktop counterpart. This means that your IP address may be seen (and logged) by Google. Note that Safe Browsing is not available for Android devices without Google Play Services.
```
Perhaps this warning is too verbose for an embedded footnote. Any feedback is welcome.

**Review by redoomed1 — CHANGES_REQUESTED** [review]

**Review by jonaharagon — CHANGES_REQUESTED** [review]
Is this text different between iOS and Android, or what?

File: docs/mobile-browsers.md [hunk 4]
```diff
@@ -91,10 +91,13 @@ Brave allows you to select additional content filters within the internal `brave
...
 </details>
 
-- [x] Select **Upgrade connections to HTTPS**
-- [x] Select **Always use secure connections**
+- [x] Select **Auto-redirect AMP pages**
+- [x] Select **Auto-redirect tracking URLs**
+- [x] Select **strict** under **Upgrade connections to HTTPS**
 - [x] (Optional) Select **Block Scripts** (1)
-- [x] Select **Strict, may break sites** under **Block fingerprinting**
+- [x] Select **Block cross-site cookies** under **Block Cookies**
```
[the remark this answers is not in the document]
Comment (redoomed1) [reply docs/mobile-browsers.md:98]: Same here (v1.66.110)

[the remark this answers is not in the document]
Comment (jonaharagon) [reply docs/mobile-browsers.md:98]: Checked and iOS doesn't have this setting at all (only a "block all cookies" _toggle_), so I'll just merge my suggestion.

**Review by jonaharagon — APPROVED** [review]
Thanks for the PR!

**Review by redoomed1 — APPROVED** [review]
As is, LGTM.

**Review by dngray — APPROVED** [review]

**Pull request merged**